src/test/java/net/sf/jsqlparser/expression/JsonFunctionTest.java

```diff
@@ -131,24 +126,24 @@ public void testArrayAgg() throws JSQLParserException {
     @Test
     public void testObject() throws JSQLParserException {
         TestUtils.assertSqlCanBeParsedAndDeparsed(
-                "SELECT JSON_OBJECT( KEY foo VALUE bar, KEY foo VALUE bar) FROM dual ", true);
-        TestUtils.assertSqlCanBeParsedAndDeparsed("SELECT JSON_OBJECT( foo:bar, foo:bar) FROM dual ",
+                "SELECT JSON_OBJECT( KEY 'foo' VALUE bar, KEY 'foo' VALUE bar) FROM dual ", true);
```

Comment:
So the test before without those quotes was wrong?

Reply:
The documentation is scarce and Oracle mentions as `String` explicitly and all the examples show `KEY` as a `String`.
I do not use those functions myself and only implemented them, when they collided with the special `StringFunctions` and hung the parser.